src/App.tsx

```diff
@@ -15,48 +16,125 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum Sortfield {
+  SORT_FILED_ALPHABET = 'Sort alphabetically',
+  SORT_FILED_LENGTH = 'Sort by length',
+}
+
+interface FilterParams {
+  sortField: Sortfield | string;
```

Comment:
this type doesn't restrict your sort field value since I can put here any string. it is better to add to enum value for default sorting.